src/main/java/org/isf/sms/providers/skebby/model/SckebbySmsRequest.java

```diff
@@ -0,0 +1,101 @@
+package org.isf.sms.providers.skebby.model;
+
+import java.io.Serializable;
```

Comment:
Imports should be placed after the license header.  Thanks

Reply:
I corrected as you suggested.